stdlib/Random/src/Random.jl

```diff
@@ -313,12 +313,26 @@ include("XoshiroSimd.jl")
 Pick a random element or array of random elements from the set of values specified by `S`;
 `S` can be
 
-* an indexable collection (for example `1:9` or `('x', "y", :z)`),
-* an `AbstractDict` or `AbstractSet` object,
+* an indexable collection (for example `1:9` or `('x', "y", :z)`)
+
+* an `AbstractDict` or `AbstractSet` object
+
 * a string (considered as a collection of characters), or
-* a type: the set of values to pick from is then equivalent to `typemin(S):typemax(S)` for
-  integers (this is not applicable to [`BigInt`](@ref)), to ``[0, 1)`` for floating
-  point numbers and to ``[0, 1)+i[0, 1)`` for complex floating point numbers;
+
+* a type (and corresponding domain) from the list below
+
+  + `typemin(S):typemax(S)` for concrete integer types (excepting [`BigInt`](@ref) which is not supported)
```

Comment:
i find this wording confusing: 

> a type (and corresponding domain) from the list below

this implies that the list below will have types as items, but items are instead the domain...

Reply:
how about reversing the order of the items? so that would be e.g.

> a type from the list below, which samples uniformly over that type's corresponding domain

* concrete integer types `T` sample from `typemin(T):typemax(T)` (excepting `BigInt`...)
* concrete floating point types sample from `[0, 1)`
* concrete complex types `Complex{T}` sample from `rand(T) + rand(T) * im`
* `<:AbstractChar` types sample from all valid Unicode scalars